src/test/resources/soop_sst_trajectory.xml

```diff
@@ -6,68 +6,849 @@
+    </attribute>
+    <attribute name="project_acknowledgement">
+      <value>'The collection of this data was funded by IMOS'</value>
+    </attribute>
+  </globalattributes>
+  <variables>
     <variable>
-      <name>id</name>
+      <name>trajectory</name>
       <encoder>integer</encoder>
       <attributes>
-        <attribute name="_FillValue" value="99999"/>
+        <attribute name="long_name" value="Unique identifier for each feature instance"/>
+        <attribute name="cf_role" value="trajectory_id"/>
       </attributes>
     </variable>
-
     <variable>
       <name>TIME</name>
-      <encoder>time</encoder>
+      <encoder>time</encoder>  
       <dimensions>
+      <!--  SO i am  following what's in the anmn_ts template but i  questioning the encoder thta is used here 'time'. NODC recommends time to be of type double, which is also the current IMOS convention. Why using type 'time' then (except the fact that TIME  is a timestamp in the DB). TIME should be converted back to double to conform the NODC format? Am i missing something here?
+      -->
         <dimension name="TIME"/>
       </dimensions>
       <attributes>
-        <attribute name="_FillValue" value="99999."/>
+        <attribute name="standard_name" value="'time'"/>
+        <attribute name="long_name" value="'time'"/>
         <attribute name="units" value="'days since 1950-01-01 00:00:00 UTC'"/>
+        <attribute name="calendar" value="'gregorian'"/>
+        <attribute name="axis" value="'T'"/>
+        <attribute name="_FillValue" value="-999999."/>   
+        <attribute name="valid_min" value="0."/>
+        <attribute name="valid_max" value="90000."/>
+        <attribute name="ancillary_variables" value="'TIME_quality_control'"/>
       </attributes>
     </variable>
     <variable>
+      <name>TIME_quality_control</name>
+      <encoder>char</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+       <attributes>
+        <attribute name="standard_name" value="'time status_flag'"/>
+        <attribute name="long_name" value="'quality flag for time'"/>
+        <attribute name="quality_control_set" value="3."/>
+        <attribute name="quality_control_conventions" value="'BOM (SST and Air-Sea flux) quality control procedure'"/>
+        <attribute name="flag_values" value="'B, C, D, E, F, G, H, J, K, L, M, Q, S, T, U, V, X, Z'"/>
+        <attribute name="flag_meaning" value="'Value_out_of_bounds Time_not_sequential Failed_T_Tw_Td_test Failed_true_wind_recomputation_test Platform_velocity_unrealistic Value_exceeds_threshold Discontinuity Erroneous_value Suspect_value_(visual) Value_located_over_land Instrument_malfunction Pre-flagged_as_suspect Spike_in_data_(visual) Time_duplicate Suspect_value_(statistical) Step_in_data_(statistical) Spike_in_data_(statistical) Value_passed_all_tests'"/>
```

Comment:
@smancini we are using the netcdfgenerator to have the opportunity to output NetCDF files consistently. It's a shame if we use a different flag scale than the IMOS one here... How about storing IMOS flags in the database? Mapping would be performed by harvester.